setup/docker/compose.yml

```diff
@@ -0,0 +1,80 @@
+networks:
+  hostnet:
+    ipam:
+      driver: default
+      config:
+        - subnet: 10.5.5.0/24
+
+volumes:
+  pulsarconf:
+  pulsardata:
+  zkdata:
+  zklogs:
+  zkdatalog:
+
+services:
+  web:
+    image: vardhi.docker.registry/varadhi:latest
+    profiles: [ "test", "service" ]
+    networks:
+      hostnet:
+        ipv4_address: 10.5.5.2
+    ports:
+      - 8080:8080
+    restart: "no"
+    depends_on:
+      pulsar:
+        condition: service_healthy
+      zookeeper:
+        condition: service_healthy
+
+  pulsar:
+    image: apachepulsar/pulsar:3.0.0
+    profiles: [ "test", "dev", "pulsar" ]
```

Comment:
what is these profiles used for?

Reply:
grouping, containers under that profile would be brought up as part of compose up/down. (docker compose --profile)